apps/search/js/providers/suggestions.js

```diff
@@ -24,46 +26,26 @@
 
     remote: true,
 
-    init: function(config) {
+    init: function() {
       Provider.prototype.init.apply(this, arguments);
-      eme.init();
     },
 
     click: function(e) {
-      var suggestion = e.target && e.target.dataset.suggestion;
-      if (suggestion) {
-       Search.setInput(suggestion);
-      }
+      var suggestion = e.target.dataset.suggestion;
+      var url = encodeTerms(SearchProvider('searchUrl'), suggestion);
+      Search.navigate(url);
     },
 
     search: function(input) {
-      return new Promise((resolve, reject) =>
-        eme.init().then(() => {
-        this.clear();
-        if (!eme.api.Search) {
-          reject();
-          return;
-        }
-
-        this.request = eme.api.Search.suggestions({
-          'query': input
-        });
-
-        this.request.then((data) => {
-          var items = data.response;
-          if (items && items.length) {
-            // The E.me API can return the query as a suggestion
-            // Filter this out
-            var matchingIndex = items.indexOf(input);
-            if (matchingIndex !== -1) {
-              items.splice(matchingIndex, 1);
-            }
-          }
-          resolve(items);
-        }, (reason) => {
-          reject();
+      return new Promise((resolve, reject) => {
+        var url = encodeTerms(SearchProvider('suggestUrl'), input);
+        LazyLoader.getJSON(url, true).then(result => {
+          var results = result[1];
+          // We add an item to search the entered term as well
+          results.unshift(result[0]);
```

Comment:
Sometimes the provider will give the entered term as a suggestion so there will be a duplicate. Not sure if this is a problem? (e.g. If I enter "abcd" into Google I get "abcd" listed twice)